src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/.template.config/template.json

```diff
@@ -1,21 +1,23 @@
 {
+  "$schema": "http://json.schemastore.org/template",
   "author": "Microsoft",
   "classifications": [
     "Web",
     "Blazor",
     "WebAssembly"
   ],
+  "name": "Blazor WebAssembly App",
   "defaultName": "WebApplication",
   "description": "A project template for creating a Blazor app that runs on WebAssembly and is optionally hosted by an ASP.NET Core app. This template can be used for web apps with rich dynamic user interfaces (UIs).",
   "groupIdentity": "Microsoft.Web.Blazor.Wasm",
+  "precedence": "6001",
```

Comment:
I set this based on to the server template.json which has a `precedence: "6000"`. Not entirely sure if it's correct or necessary.

Reply:
Long story short the `precedence` property is used to decide which template to use if more than one is available. We generally [increment it by 1000](https://github.com/aspnet/AspNetCore/commit/92f771f1171e521e3836cef85bf87441568acbe0#diff-b75849653d13645bd435dd1e8883ea39R11) when there's an update of either major or minor version (e.g. 3.0->3.1).

Given that I suggest setting it here to `"precedence": "6001",`